test/e2e/storage/regional_pd.go

```diff
@@ -189,19 +190,19 @@ func testZonalFailover(c clientset.Interface, ns string) {
 	framework.ExpectNoError(err)
 
 	defer func() {
-		framework.Logf("deleting statefulset%q/%q", statefulSet.Namespace, statefulSet.Name)
+		e2elog.Logf("deleting statefulset%q/%q", statefulSet.Namespace, statefulSet.Name)
 		// typically this claim has already been deleted
 		framework.ExpectNoError(c.AppsV1().StatefulSets(ns).Delete(statefulSet.Name, nil /* options */),
 			"Error deleting StatefulSet %s", statefulSet.Name)
 
-		framework.Logf("deleting claims in namespace %s", ns)
+		e2elog.Logf("deleting claims in namespace %s", ns)
 		pvc := getPVC(c, ns, regionalPDLabels)
 		framework.ExpectNoError(c.CoreV1().PersistentVolumeClaims(pvc.Namespace).Delete(pvc.Name, nil),
 			"Error deleting claim %s.", pvc.Name)
 		if pvc.Spec.VolumeName != "" {
 			err = framework.WaitForPersistentVolumeDeleted(c, pvc.Spec.VolumeName, framework.Poll, pvDeletionTimeout)
 			if err != nil {
-				framework.Logf("WARNING: PV %s is not yet deleted, and subsequent tests may be affected.", pvc.Spec.VolumeName)
+				e2elog.Logf("WARNING: PV %s is not yet deleted, and subsequent tests may be affected.", pvc.Spec.VolumeName)
```

Comment:
@draveness another usecase where exposing the log level in `e2elog` could be useful.